SourceryRuntime/Sources/Class.swift

```diff
@@ -24,7 +24,8 @@ import Foundation
                          typealiases: [Typealias] = [],
                          attributes: [String: Attribute] = [:],
                          annotations: [String: NSObject] = [:],
-                         isGeneric: Bool = false) {
+                         isGeneric: Bool = false,
+                         genericTypes: [Generic] = []) {
```

Comment:
I think `genericParameters` is a name that is more aligned with Swift terminology